### PR TITLE
Add github app installation instructions

### PR DESCRIPTION
**Pull request by welteki**
Signed-off-by: Han Verstraete (OpenFaaS Ltd) <han@openfaas.com>

## Description

Add github app installation instructions

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
- [ ] My issue has received approval from the maintainers or lead with the `design/approved` label

## How Has This Been Tested?

Verified locally

Screenshot 2022-09-26 at 14 08 31

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/registration.md [hunk 1]
```diff
@@ -18,6 +18,15 @@ Once accepted into the pilot program, your an administrator from your GitHub org
...
 
+1. Open the link that you received to install our GitHub App
+2. Click **Install app**
+3. Select the organisation you want to install the Actuated app to
+4. Install the app on all repositories or select repositories
+
+    ![Install GitHub app](/images/install_github_app.png)
+
+5. Once installed you will will see the permissions and other configuration options for the Actuated app on your selected account. To install the app on another account you can repeat these installation steps.
+
```
[the remark this answers is not in the document]
Comment (welteki) [reply docs/registration.md:29]: You mean a link to the GitHub documentation for this?

[the remark this answers is not in the document]
Comment (alexellis) [reply docs/registration.md:29]: Sounds fine yes

**Comment by alexellis** [comment]
Please rebase

File: docs/registration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (welteki) [reply docs/registration.md]: I added instructions to show were to uninstall or manage the app.

**Comment by welteki** [comment]
Rebased and included instructions to show were to uninstall or manage the app.

**Pull request merged**